test/functional/services/lib/leadfoot_element_wrapper/leadfoot_element_wrapper.js

```diff
@@ -339,4 +340,18 @@ export class LeadfootElementWrapper {
   async scrollIntoViewIfNecessary() {
     await this._leadfoot.execute(scrollIntoViewIfNecessary, [this._leadfootElement, this._fixedHeaderHeight]);
   }
+
+  /**
+   * Gets element innerHTML and wrap it up with cheerio
+   *
+   * @nonstandard
+   * @return {Promise<void>}
+   */
+  async getInnerHtml() {
```

Comment:
I've been trying to think of a better name for this since it's exactly returning the HTML, what about `inspectDom()`, `loadHtmlElements()`, `parseDomContent()`?

Reply:
I agree, changing it to `parseDomContent`